Tests/XCUITests/DisplaySettingsTests.swift

```diff
@@ -48,5 +51,7 @@ class DisplaySettingTests: BaseTestCase {
         let switchValueAfter = app.switches["SystemThemeSwitchValue"].value!
         XCTAssertEqual(switchValueAfter as! String, "1")
         XCTAssertFalse(app.tables["DisplayTheme.Setting.Options"].otherElements.staticTexts["SWITCH MODE"].exists)
+        mozWaitForElementToNotExist(app.tables["DisplayTheme.Setting.Options"].otherElements.staticTexts["SWITCH MODE"])
```

Comment:
Not sure this makes sense after the XCTAssertFalse... either we add it before or we remove the XCTAssertFalse... 
That condition will fail if the element is there and so the mozWaitForElementToNotExist will not run

Reply:
Actually the intention was to add another XCTAssertFalse for "THEME PICKER". Updated the test.